src/core/agent/services/connectionService.ts

```diff
@@ -218,10 +217,23 @@ class ConnectionService extends AgentService {
     alias?: string,
     groupId?: string
   ): Promise<string> {
-    const result = await this.props.signifyClient
-      .oobis()
-      .get(signifyName, ConnectionService.DEFAULT_ROLE);
+    const result = await this.props.signifyClient.oobis().get(signifyName);
+    if (!result.oobis[0]) {
+      return "";
+    }
     const oobi = new URL(result.oobis[0]);
+    const identifier = await this.props.signifyClient
+      .identifiers()
+      .get(signifyName);
+    //This condition is used for multi-sig oobi
+    if (identifier && identifier.group) {
+      const pathName = oobi.pathname;
+      const agentIndex = pathName.indexOf("/agent/");
+      if (agentIndex !== -1) {
+        oobi.pathname = pathName.substring(0, agentIndex);
+        return oobi.toString();
```

Comment:
Does this just strip off the agent AID at the end? I thought we had to insert our own one with a replace

Reply:
No, we need to cut it. Even if we replace it with our own, it won't work as we expected.